DOCSP-39069: Add docs release notes for w/e May 3 (#757)
mongodb/docs-app-services: docs-release-notes.md

# May 3, 2024

## Edge Server

> - Landing Page
>   - Update details for Public Preview. Add info about using MongoDB Drivers and tools. Remove verbiage or clarify behavior that is Atlas Device SDK-specific.
>   - Landing Page: Add marketing CTA and link to Public Preview announcement blog entry.

Reviewer: ```suggestion
- Landing Page:
  - Update details for Public Preview. Add information about using MongoDB Drivers and tools, and update existing information about Atlas Device SDK-specific behavior.
  - Add link to Public Preview announcement blog entry.
```

- Manage Edge Servers: New page describing how to manage Edge Servers in Atlas.

> - Configure Edge Server

Reviewer: ```suggestion
- Configure Edge Server:
```

  - Update the page for the Public Preview. Add `edgectl` commands and build out new sections with additional details.
  - Add information about managing `edgectl` profiles with the various `profile` commands.
  - New "Simulate Offline Connectivity" section describing how to use offline demo mode.

> - Connect to an Edge Server: Update the page for the Public Preview. Add details about Roaming with Atlas Device SDK, Connecting via the wireprotocol with authentication, and Connecting via Compass.

Reviewer: ```suggestion
- Connect to an Edge Server: Update the page for the Public Preview. Add details about roaming with Atlas Device SDK, connecting with the Wire Protocol with authentication, and connecting with Compass.
```

- Edge Server MongoDB API Support: Add `findAndModify` command to update options.

> - Edge Server Permissions Guide: New page providing a high-level overview of Edge Server permissions, and how they relate to client permissions.

Reviewer: ```suggestion
- Edge Server Permissions Guide: New page providing a high-level overview of Edge Server permissions and how they relate to client permissions.
```

- Monitor Edge Server: New page with information about Edge Server logs, metrics, and status.

## Reference

- Admin API: New Edge Server endpoints:
  - List Edge Server instances
  - Get information about an Edge Server instance
  - Create an Edge Server instance
  - Delete an Edge Server instance
  - Regenerate an Edge Server registration key
  - Rename an Edge Server instance

> - Billing: New "Atlas Edge Server" section with a high-level blurb about billing.

Reviewer: ```suggestion
- Billing: New "Atlas Edge Server" section with a high-level information about billing while in Public Preview.
```
